build/AzurePipelineTemplates/CsWinRT-Benchmarks-Steps.yml

```diff
@@ -82,13 +58,4 @@ steps:
 
 # Component Detection
 - task: ComponentGovernanceComponentDetection@0

```

Comment:
I'm not sure a manual ComponentGovernance step is needed when running in OneBranch.

Reply:
Yea I wasn't sure if I should keep this or not given this was the one running on Azure pipelines rather than OneBranch.